pkg/infoschema/test/clustertablestest/cluster_tables_test.go

```diff
@@ -1776,14 +1777,14 @@ func TestMDLViewIDConflict(t *testing.T) {
 	wg := &sync.WaitGroup{}
 	wg.Add(1)
 	go func() {
-		ddlTK1.MustExec("ALTER TABLE t ADD COLUMN b INT;")
+		ddlTK1.MustExec("ALTER TABLE t ADD index(a);")
```

Comment:
now the routine will hold worker until it exist, so we change to reorg job to make this test work